mountpoint-s3/src/metrics/data.rs

```diff
@@ -50,13 +50,7 @@ impl Metrics {
                         .join(",")
                 )
             };
-            info!(
-                target: METRICS_TARGET_NAME,
-                "{}{}: {}",
-                key.name(),
-                labels,
-                metric,
-            );
+            info!(target: METRICS_TARGET_NAME, "{}{}: {}", key.name(), labels, metric,);
```

Comment:
Changes in `data.rs` and `fs.rs` are due to `make fmt`. Not sure if I should keep it?

Reply:
Try reverting them. `make fmt` does not seem to be the cause to me.